pkg/controller/controller.go

```diff
@@ -1131,6 +1131,7 @@ func (s *Service) ControllerGetCapabilities(_ context.Context, _ *csi.Controller
 	if s.isHealthMonitorEnabled {
 		for _, capability := range []csi.ControllerServiceCapability_RPC_Type{
 			csi.ControllerServiceCapability_RPC_GET_VOLUME,
+			csi.ControllerServiceCapability_RPC_LIST_VOLUMES,
 			csi.ControllerServiceCapability_RPC_LIST_VOLUMES_PUBLISHED_NODES,
 			csi.ControllerServiceCapability_RPC_VOLUME_CONDITION,
 		} {
```

Comment:
I see that ListSnapshots() returns snapshots of both block volumes and file systems, while ListVolumes() return only volumes. Should this call include file systems as well? 
@AkshaySainiDell @adarsh-dell

Reply:
After clarification with @adarsh-dell it looks like it should return file systems as well. But this is out of scope of this defect. As discussed with @alikdell just this particular capability needs to be published for now.

Reply:
Yes, please add what is missing at krv-18102